system_test/helpers/aest_nodes.erl

```diff
@@ -1,5 +1,7 @@
 -module(aest_nodes).
 
+-include_lib("eunit/include/eunit.hrl").
```

Comment:
Do you mean `-include_lib("stdlib/include/assert.hrl").` - for `?assertEqual`? Anyway this file is test framework so not an issue.